commitzinho da tela 010

WalterHenri opened the pull request:
aceite ai va.

WalterHenri committed:
commitzinho

Gabrielx47 reviewed (changes requested):
O títtulo "Data"  poderia está ALINHADO A ESQUERDA, acredito que ficaria mais coerente.

WalterHenri committed:
tela 010 conteudo alinhado

antes a data e a frequência estavam fora do alinhamento convencional trazendo uma imagem desagradável para a tela em questão.

Gabrielx47 reviewed (approved):
Bom trabalho, o alinhamento ao centro ficou perfeito.

joannestephany reviewed (approved):
Aprovado

joannestephany commented:
Na real na real, aprovei mas o símbolo da samu em azul ainda me deixa em dúvida
